style.css

```diff
@@ -248,17 +273,47 @@ h3{
 
 /*style for my paragraphs*/
 p{
-	font-family: 'Poppins', sans-serif;
+	font-family: 'Lexend', sans-serif;
 	font-weight: 400;
 	color: black;
-	font-size: 11pt;
+	font-size: 1.1vw;
 	letter-spacing: 0.01em;
 }
 
 
 /*style for footer*/
 .footer{
+	display: grid;
```

Comment:
used responsive grid system for footer display to keep everything in their respective sections